### ISSUE TITLE
Which points are optimum?

### ISSUE THREAD

**Issue by vvandermeersch** [vvandermeersch/synchrony#6]
@lizzieinvancouver I'm opening this question in case you have an opinion!
So far, I've defined the optimal days as those with the optimality in the top 5% (i.e. >95th percentile), but it would be possible to have a more or less restrictive definition. Here's two random sites, one in Spain, one in Sweden:
<img src="https://github.com/user-attachments/assets/b149cf83-acb2-4859-8668-1d210b65a26c" width="550">

**Comment by lizzieinvancouver** [comment]
@vvandermeersch I have mixed feelings! I'd like to write it up soon and not complicate it too much so I err towards sticking with what you have done, but I also am intrigued by whether the optimal period might be pretty long in some places (e.g., your southern site). I think this means -- it's really up to you and I will be supportive of either option!

**Comment by vvandermeersch** [comment]
@lizzieinvancouver I agree that it could be nice to finalize it soon (but it would still not be complicated to take Q97.5 or Q90 if you prefer). Anyway, we can keep in mind that there are still things to dig around these "optimum stability-related questions" (including a null model)... for later work?

**Comment by lizzieinvancouver** [comment]
>  Anyway, we can keep in mind that there are still things to dig around these "optimum stability-related questions" (including a null model)... for later work?

@vvandermeersch Sounds great!
